website/docs/docs/use-dbt-semantic-layer/exports.md

```diff
@@ -0,0 +1,193 @@
+
+```bash
+dbt sl export --saved-query sq_name --select export_1,export2
+```
+
+<details>
+<summary>Overriding export configurations</summary>
+
+The `--select` flag is mainly used to include or exclude specific exports. If you need to change these settings, you can use the following flags to override export configurations:
+
+- `--export-as` &mdash; Defines the materialization type (table or view) for the export. This creates a new export with its own settings and is useful for testing in development.
+- `--schema` &mdash;  Specifies the schema to use for the  materialized table or view.
+- `--alias` &mdash; Assigns a custom alias to the materialized table or view. This overrides the default export name.
+
+Be careful. The `--select` flag _can't_ be used with `alias` or `schema`.
+
+For example, you can use the following command to create a new export named `new_export` as a table:
+
+```bash
+dbt sl export --saved-query sq_number1 --export-as table --alias new_export
+```
+</details>
+
+### Exports in production
+
+To enable exports in production using the dbt Cloud job scheduler, follow these steps:
+
+**Set environment variable**
+
+1. Go to your **Environments** page by clicking **Deploy** in the navigation.
+2. Select **Environment variables**
+3. [Set the environment variable](/docs/build/environment-variables#setting-and-overriding-environment-variables) key to `DBT_INCLUDE_SAVED_QUERY` and the environment variable(s) value to `TRUE` (`DBT_INCLUDE_SAVED_QUERY=TRUE`).
+
+Doing this ensures saved queries and exports are included in your dbt build job. For example, running `dbt build sq_name` runs the equivalent of `dbt sl export --saved-query sq_name` in the dbt Cloud Job scheduler.
+
+If exports aren't needed, you can set the value(s) to `FALSE` (`DBT_INCLUDE_SAVED_QUERY=FALSE`).
+
+<Lightbox src="/img/docs/dbt-cloud/semantic-layer/deploy_exports.jpg" width="90%" title="Add an environment variable to run exports in your production run." />
+
+When you run a build job, any saved queries downstream of the dbt models in that job will also run. To make sure your export data is up-to-date, run the export as a downstream step (after the model).
+
+**Create and execute exports**
+
+1. Create a [deploy job](/docs/deploy/deploy-jobs) and ensure the `DBT_INCLUDE_SAVED_QUERY=TRUE` environment variable is set, as described earlier. 
+   - This enables you to run any export that needs to be refreshed after a model is build.
+2. After dbt finishes building the models, the MetricFlow Server processes the exports, compiles the necessary SQL, and executes this SQL against your data platform.
+3. Review the exports execution details in the jobs logs and confirm the export was run successfully. This helps troubleshoot or ensure accuracy. Since saved queries are integrated into the dbt DAG, all outputs related to exports are available in the job logs.
```

Comment:
```suggestion
3. Review the exports' execution details in the jobs logs and confirm the export was run successfully. This helps troubleshoot and to ensure accuracy. Since saved queries are integrated into the dbt DAG, all outputs related to exports are available in the job logs.
```